### DISCUSSION TITLE
Additional Game Details property on Game page and table

### DISCUSSION THREAD

**Discussion by ScottFromDerby** [RetroAchievements/RAWeb#708] [Ideas]
Would be really useful to have a developer-only editable text box on a game page to allow developers or above to clarify any nuances or considerations for the game/set.

**Comment by kdecks** [comment]
As in a dev read-me? Great idea

**Comment by meleu** [comment]
This feature request looks duplicated. Salsa said something similar on #45

**EDIT**: Closed #45 and left this here open.

**Comment by televandalist** [comment]
"Devs Beware: This game's memory is horrendous. Good luck."

🤔
